README.md

```diff
@@ -35,6 +35,10 @@
 
 ---
 
+> [!WARNING]
+> This is the working branch for the work in progress 2.0 SDK, see [this tracking issue](https://github.com/open-telemetry/opentelemetry-js/issues/5148) for details.
```

Comment:
```suggestion
> This is the working branch for the work in progress on 2.0 SDK, see [this tracking issue](https://github.com/open-telemetry/opentelemetry-js/issues/5148) for details.
```

or `for`. It just feels like is missing a word here

Reply:
Hmm, it seems correct to me. 🤔 
@dyladan used the wording in https://github.com/open-telemetry/opentelemetry-js/pull/4226/files but I might've messed up the grammar with my changes 😅 

English is difficult 😞